src/pages/NewAppointment.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable no-console */
+import React, { useEffect, useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchReserves, postReserve } from '../redux/Reserve/reserveSlice';
+import { fetchTherapists } from '../redux/Therapy/therapySlice';
+import '../css/NewAppointment.css';
+import specializationArr from '../components/specialization';
+import backImg from '../img/back.png';
+import fadingBreak from '../img/fading_break.png';
+import logoImg from '../img/logo.png';
+
+function NewAppointment() {
```

Comment:
- [ ] For some reason, If I try to create or view my appointments on the mobile view, it will send me right to the login page:

![image](https://github.com/MAleemH/final-capstone-front-end/assets/57421823/2a362c65-a709-4315-be68-ad17c837f9e9)

This renders me unable to view or create appointments on the phone view.  Kindly fix this bug so your app works as intended 👍